README.md

```diff
@@ -1,4 +1,18 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four neighbouring blocks. Currently, however, the module is a little bit bugged. This means that the fences do not connect correctly the very moment they are placed. The issue can be fixed by forcing an update for given fence blocks, for instance by placing and mining arbitrary block right next to the fence. Another problem this module currently has is that the ground right around fence is severely darkened if you have SSAO enabled in your graphics settings. This module also lacks any sort of gate, although it used to have it in past.
+
+![Fences showcase](https://i.imgur.com/5cT16lG.png)
+
+# Getting started
+
+If you want to start playing with fences, you have currently one single option:
```

Comment:
Change this to "To start playing with fences, follow these steps:" or something like that.